.pre-commit-config.yaml

```diff
@@ -34,7 +34,8 @@ repos:
       - id: check-merge-conflict
       - id: check-symlinks
       - id: check-vcs-permalinks
-      #- id: check-yaml
+      - id: check-yaml
+        exclude: ^plugins/integrations/kubernetes-service/src/main/resources/conf
```

Comment:
why this exclusion @jbampton ? are the yamls there not valid?

Reply:
I removed the exclusion and did a fix up of the YAML.